src/extension/css/src/main/java/org/geoserver/community/css/web/StylePanel.java

```diff
@@ -23,19 +23,24 @@
 import org.apache.wicket.model.IModel;
 import org.apache.wicket.model.PropertyModel;
 import org.geoserver.catalog.StyleInfo;
+import org.geoserver.platform.resource.Resource;
+import org.geoserver.platform.resource.Resources;
 import org.geoserver.web.wicket.CodeMirrorEditor;
 
 public class StylePanel extends Panel {
     private String styleBody;
 
     public StylePanel(String id, IModel<CssDemoPage> model, final CssDemoPage page,
-            final File cssFile) {
+            final Resource cssFile) {
         super(id, model);
-        if (cssFile != null && cssFile.exists()) {
+        if (cssFile != null && Resources.exists(cssFile)) {
+            InputStream is = cssFile.in();
             try {
-                styleBody = FileUtils.readFileToString(cssFile);
+                styleBody = IOUtils.toString(is, "UTF-8");
```

Comment:
Guys, did you check how things are written before assuming UTF-8 is the right approach? It's actually a few lines down, and it does not have any charset. Also, it means all the CSS around right now are using platform charset. Given this, isn't it better to leave it like it was before, no explicit charset setting?